src/Game.py

```diff
@@ -0,0 +1,29 @@
+from copy import deepcopy
+
+boards =	{
+  "player1": {
+    "primary": [],
+    "opponent": [],
+  },
+  "player2": {
+    "primary": [],
+    "opponent": [],
+  },
+  "currentPlayer": 0,
+  "hit": 0
```

Comment:
These two vars does not belong in boards var imo. Maybe the var is misnamed or you need an other var.